libs/sdk-ui-web-components/src/visualizations/Dashboard.tsx

```diff
@@ -0,0 +1,62 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import invariant from "ts-invariant";
+
+import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
+import { ILocale } from "@gooddata/sdk-ui";
+import {
+    CustomElementAdapter,
+    EVENT_HANDLER,
+    GET_VISUALIZATION,
+    LOAD_COMPONENT,
+} from "./CustomElementAdapter";
+
+type IDashboard = typeof import("@gooddata/sdk-ui-dashboard/esm/presentation/dashboard/Dashboard").Dashboard;
```

Comment:
Can we do
```ts
import type { Dashboard as IDashboard } from "@gooddata/sdk-ui-dashboard";
```
instead? That should also make the dependency cruiser checks that are currently failing pass.

Reply:
Looks like deps cruiser is worried about css imports, not this one...
https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/10805/console